notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "b6dbb8aa-c070-4e7d-8b7a-64588e7f0da4",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    def segment_wrapper(image, task, run_quantized=False):
</pre><p>What do you think about checkbox for <code>run_quantized</code> argument? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
We've had a similar discussion with @AlexKoff88 for the grammar correction notebook. Then decided to have a separate button for quantized model inference. I guess it should be the same way here.